services/clients/firebase_client.py

```diff
@@ -40,15 +42,19 @@ def get_collection(self, collection_name) -> Generator[DocumentSnapshot, Any, No
         docs = self.db.collection(collection_name)
         return docs.stream()
 
-    def add_blob(self, blob_name, data):
-        blob = self.bucket.blob(blob_name)
+    def add_blob(self, blob_url, data):
+        blob = self.bucket.blob(blob_url)
         blob.upload_from_string(data)
 
-    def get_blob(self, blob_name) -> Blob:
-        return self.bucket.get_blob(blob_name)
+    def get_blob(self, blob_url) -> Blob:
+        return self.bucket.get_blob(blob_url)
+    
+    def get_blob_url(self, blob_name, folder="repo") -> str:
```

Comment:
Added this just to make things easier. lmk if theres another way to do this

Reply:
For now this works well. We should review how we want to design our Cloud Storage in general. Maybe we could even store it like a repo:
`/{user_id}/{repo_id}/folder1`
`/{user_id}/{repo_id}/folder2`
`/{user_id}/{repo_id}/folder3`
`/{user_id}/{repo_id}/folder3/file.py`
`/{user_id}/{repo_id}/file1.py`
`/{user_id}/{repo_id}/file2.py`

Might not be worth it though, idk. Added the topic to our doc